src/Php56/bootstrap.php

```diff
@@ -21,4 +21,16 @@ function hash_equals($knownString, $userInput) { return p\Php56::hash_equals($kn
 
         function ldap_escape($subject, $ignore = '', $flags = 0) { return p\Php56::ldap_escape($subject, $ignore, $flags); }
     }
+
+    // Missing functions in PHP 5.5.9 - affects 32 bit builds of Ubuntu 14.04LTS
```

Comment:
I propose to enclose these checks in a `if (8 > PHP_INT_SIZE) {` (and removing it from this `PHP_VERSION_ID < 50600` block) to get a few more CPU cycles on 64b setups (which are the most common)